docs/datascientist_onboard.md

```diff
@@ -0,0 +1,235 @@
+  ```
+  docker run -it paiexample/pai.example.hdfs /bin/bash
+  ```
+  Then use the following commands to upload data:
+  ```
+  hdfs dfs -put /your_local_data_path/ hdfs://hdfs-url/hdfs_path/
+  ```
+  More commands please refer to the [HDFS commands guide](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-hdfs/HDFSCommands.html) for details.
+
+#### 1.2 Operate HDFS data
+**Note:** The relative positions of your code and the data downloaded into the container may be changed, debug your code and make sure it can get access to the dataset.
+
+**1.2.1 Small dataset**
+
+If your data size is small, you have the following ways to handle your code and data:
+
+1) You can use HDFS commands in your code to download data to the container when it is launched.
+You can specify the download commands in [job JSON configuration file](#3.-Write-a-job-json-configuration-file) `command` field. The commands will be executed right after the container is up. e.g.
+    ```
+    {
+        "jobName": "${your_job_name}",
+        "image": "${your_job_container}",
+        ...
+        ...
+        "taskRoles": [
+            {
+                ...
+                ...
+                // This command make a dir /root/img/ in this job container, download data from hdfs to this dir 
+                // and then execute job script run_job.py
+                // Replace hdfs://10.151.40.234:9000 with your own cluster's HDFS url
+                "command": "if [ ! -d \"/root/img/\" ]; then mkdir /root/img; fi && hdfs dfs -get hdfs://10.151.40.234:9000/yife/data/img/* /root/img/ && /usr/bin/python3 run_job.py"
+            }
+        ]
+    }
+    ```
+    
+    Or you can download data in your code, use the above example, e.g. you can execute `hdfs dfs -get` command in your `run_job.py`. 
+
+2) Pack code and data into job container
+
+    You can pack your code and dataset into your container when you are [building your job container](#3.1-Need-to-modify-the-DL-framework's-version,-or-add-some-dependencies).
+
+    Use `COPY` command in Dockerfile to copy files from local into container, then build it. 
+    
+    For more information please refer to [Dockerfile reference](https://docs.docker.com/engine/reference/builder/#copy).
+
+    **Note:** We don't recommend to pack code into container unless your code won't change. Because every time you change your code you will build a new image. You can upload your code to HDFS, then specify the HDFS url of your code in [job JSON configuration file](#3.-Write-a-job-json-configuration-file) `codedir` field, PAI will automatically download it to the job container's path `/pai/code/`.
+
+
+3) Use HDFS mount.    
```

Comment:
Seems `hadoop-hdfs-fuse` is another mount tools(https://github.com/Microsoft/pai/blob/27be2f781498f16f67c44ebf2cfd9199a55fdcf9/pai-management/doc/hdfs.md#mountable-hdfs-), should we keep consistent with our docs

Reply:
But our CNTK example which used HDFS mount is using https://github.com/Microsoft/hdfs-mount.git I think it's OK

Reply:
Get it